src/spaces/luv.js

```diff
@@ -0,0 +1,85 @@
+import ColorSpace from "../space.js";
+import {WHITES} from "../adapt.js";
+import xyz_d65 from "./xyz-d65.js";
+import {uv} from "../chromaticity.js";
+import {isNone, skipNone} from "../util.js";
+
+let white = WHITES.D65;
+
+const ε = 216 / 24389;  // 6^3/29^3 == (24/116)^3
+const κ = 24389 / 27;   // 29^3/3^3
+const [U_PRIME_WHITE, V_PRIME_WHITE] = uv({space: xyz_d65, coords: white});
+
+export default new ColorSpace({
+	id: "luv",
+	name: "Luv",
+	coords: {
+		l: {
+			refRange: [0, 100],
+			name: "L"
```

Comment:
@lloydk Was it intentional to accept `L` here instead of `Lightness`? See e.g. https://github.com/color-js/color.js/issues/345

Reply:
Good catch, created a [PR](https://github.com/color-js/color.js/pull/406) to change it to `Lightness`.